cmake/f3dPlugin.cmake

```diff
@@ -243,7 +243,7 @@ macro(f3d_plugin_build)
 
   list(TRANSFORM F3D_PLUGIN_VTK_MODULES PREPEND "VTK::")
 
-  target_link_libraries(f3d-plugin-${F3D_PLUGIN_NAME} PUBLIC
+  target_link_libraries(f3d-plugin-${F3D_PLUGIN_NAME} PRIVATE
```

Comment:
Why is it needed?

Reply:
actually not needed, but you mentionned that it was a mistake for it to be public

Reply:
It does not make any difference here I think since there is no linkage to plugin libraries.

Reply:
since there is no reason for it to be pulic, lets keep it private ?